### PR TITLE
Chronos: add normalization for `TCNForecaster` of tensorflow backend

### PR DESCRIPTION
**Pull request by plusbang**
### 1. Why the change?

similar reason as https://github.com/intel-analytics/BigDL/pull/6405. Experiment results on datasets (electricity and nyc_taxi) demonstrate the effectiveness of normalization extensions.

### 2. User API changes

a new parameter `normalization` when initiate TCNForecaster
```python
forecaster = TCNForecaster(past_seq_len = 96,
                           future_seq_len = 96,
                           input_feature_num = 1,
                           output_feature_num = 1,
                           normalization=True)
```

### 3. Summary of the change 

Add normalization model wrapper for TCN-keras.

### 4. How to test?
- [x] Unit test

### CODE REVIEW

**Review by TheaperDeng — APPROVED** [review]
LGTM
